docs/algorithms/sig/dilithium.md

```diff
@@ -12,7 +12,7 @@
 - **Optimized Implementation sources**: https://github.com/pq-crystals/dilithium/commit/3e9b9f1412f6c7435dbeb4e10692ea58f181ee51 with copy_from_upstream patches
   - **pqclean-aarch64**:<a name="pqclean-aarch64"></a>
       - **Source**: https://github.com/PQClean/PQClean/commit/66e50172055aaf1b9a16d8f35fe03b0807f2723e with copy_from_upstream patches
-      - **Implementation license (SPDX-Identifier)**: (CC0-1.0 or Apache-2.0) and (CC0-1.0 or MIT) and MIT
+      - **Implementation license (SPDX-Identifier)**: CC0-1.0 and (CC0-1.0 or Apache-2.0) and (CC0-1.0 or MIT) and MIT
```

Comment:
Do I understand this logical statement to now mean that CC0 applies for PQClean code for Dilithium `aarch64` (as well as --the commercially less problematic-- MIT license)? In other words, from the perspective of #1437 an automated license check permitting only MIT and Apache code now has to exclude Dilithium `aarch64` code (which was not the case before this commit), right?

Reply:
I'd personally agree with this interpretation, but tagging @beldmit for his feedback.
The CC0 was added in the commit because the main LICENSE file mentions only CC0, I've added an issue upstream to clarify if this is an oversight or intentional: https://github.com/PQClean/PQClean/issues/495.